Games/types/Mafia/roles/cards/CanVisitSelf.js

```diff
@@ -0,0 +1,13 @@
+const Role = require("../../Card");
+
+module.exports = class CanVisitSelf extends Card {
+    constructor(role) {
+        super(role);
+
+    this.meetingMods = {
+      "*": {
+        targets: { include: ["alive"], exclude: [] },

```

Comment:
canvisitself is not an appropriate name, 

e.g. roles like hooker currently visit non-mafia only

canvisitself + hooker implies it's non-mafia + self, when here it's everyone

suggestion: "can visit everyone"

Reply:
nvm, the whole card/modifier is superfluous anyway. will remove; rest of PR should be ready